pkg/services/rendering/rendering.go

```diff
@@ -110,6 +111,20 @@ func (rs *RenderingService) RenderErrorImage(err error) (*RenderResult, error) {
 }
 
 func (rs *RenderingService) Render(ctx context.Context, opts Opts) (*RenderResult, error) {
+	startTime := time.Now()
+	result, err := rs.render(ctx, opts)
+	elapsedTime := time.Since(startTime).Milliseconds()
+	if err != nil {
+		metrics.MRenderingRequestTotal.WithLabelValues("error").Inc()
+		metrics.MRenderingSummary.WithLabelValues("error").Observe(float64(elapsedTime))
```

Comment:
Not sure if this makes more sense?

```suggestion
		metrics.MRenderingRequestTotal.WithLabelValues("failure").Inc()
		metrics.MRenderingSummary.WithLabelValues("failure").Observe(float64(elapsedTime))
```